server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java

```diff
@@ -64,6 +68,10 @@ public void configure(Binder binder)
       JettyBindings.addQosFilter(binder, "/druid/worker/v1/chat/*", maxRequests);
     }
 
+    Multibinder.newSetBinder(binder, ServletFilterHolder.class)
+               .addBinding()
+               .to(JettyResponseHeaderFilterHolder.class);
```

Comment:
can you make this a class that extends `JettyResponseHeaderFilterHolder` instead? that way someone else can use `JettyResponseHeaderFilterHolder`.

I actually don't really know for certain how it would behave if multiple guice modules bind the multimap to `JettyResponseHeaderFilterHolder`